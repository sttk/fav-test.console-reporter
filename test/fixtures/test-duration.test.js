'use strict';

describe('durations', function() {
  this.slow(10000);

  describe('when slow', function() {
    it('should highlight in red', function(done) {
      this.slow(200);
      setTimeout(done, 201);
    });
  });

  describe('when reasonable', function() {
    it('should highlight in yellow', function(done) {
      this.slow(400);
      setTimeout(done, 201);
    });
  });

  describe('when fast', function() {
    it('should highlight in green', function(done) {
      setTimeout(done, 200);
    });
  });
});

