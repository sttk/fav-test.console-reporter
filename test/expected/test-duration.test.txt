

  durations
    when slow
      ✓ should highlight in red (202ms)
    when reasonable
      ✓ should highlight in yellow (204ms)
    when fast
      ✓ should highlight in green


  3 passing (615ms)

